### PR TITLE
Integrate system capabilities

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**

### CODE REVIEW

**Comment by LuxoftAKutsan** [comment]
Rebuild required

**Comment by LuxoftAKutsan** [comment]
@dev-gh @AKalinockLuxoft please review

File: src/components/application_manager/src/commands/hmi/rc_get_capabilities_response.cc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+RCGetCapabilitiesResponse::RCGetCapabilitiesResponse(
+    const MessageSharedPtr& message, ApplicationManager& application_manager)
+    : ResponseFromHMI(message, application_manager) {}
+
+RCGetCapabilitiesResponse::~RCGetCapabilitiesResponse() {}
+
+void RCGetCapabilitiesResponse::Run() {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  HMICapabilities& hmi_capabilities = application_manager_.hmi_capabilities();
```
Comment (AKalinich-Luxoft) [inline src/components/application_manager/src/commands/hmi/rc_get_capabilities_response.cc]: @LuxoftAKutsan this line also could be moved inside if at line 50

File: src/components/application_manager/src/commands/mobile/get_system_capability_request.cc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    return;
+  }
+
+  if ((*message_)[strings::msg_params].empty()) {
+    LOG4CXX_ERROR(logger_, strings::msg_params << " is empty.");
+    SendResponse(false, mobile_apis::Result::INVALID_DATA);
+    return;
+  }
+  smart_objects::SmartObject response_params(smart_objects::SmartType_Map);
+  // response_params[strings::system_capability][strings::system_capability_type]
```
Comment (AKalinich-Luxoft) [inline src/components/application_manager/src/commands/mobile/get_system_capability_request.cc]: @LuxoftAKutsan please remove this comment

File: src/components/application_manager/src/commands/mobile/get_system_capability_request.cc [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  // = (mobile_apis::SystemCapabilityType::PHONE_CALL);
+  mobile_apis::SystemCapabilityType::eType response_type =
```
Comment (AKalinich-Luxoft) [inline src/components/application_manager/src/commands/mobile/get_system_capability_request.cc]: @LuxoftAKutsan can be const

File: src/components/application_manager/test/commands/hmi/rc_is_ready_request_test.cc [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+class RCIsReadyRequestTest
+    : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
+  RCIsReadyRequestTest() : command_(CreateCommand<RCIsReadyRequest>()) {}
+
+  void SetUpExpectations(bool is_rc_cooperating_available,
+                         bool is_send_message_to_hmi,
+                         bool is_message_contain_param,
+                         am::HmiInterfaces::InterfaceState state) {
```
Comment (AKalinich-Luxoft) [inline src/components/application_manager/test/commands/hmi/rc_is_ready_request_test.cc]: @LuxoftAKutsan all params could be const

File: src/components/remote_control/include/remote_control/rc_module_constants.h [hunk 5]
```diff
@@ -35,7 +35,13 @@
 
 namespace remote_control {
 
-namespace strings {}  //  strings
+namespace strings {
+// RemoteControlCapabilities constants
+const char kclimateControlCapabilities[] = "climateControlCapabilities";
+const char kradioControlCapabilities[] = "radioControlCapabilities";
+const char kbuttonCapabilities[] = "buttonCapabilities";
```
Comment (AKalinich-Luxoft) [inline src/components/remote_control/include/remote_control/rc_module_constants.h]: @LuxoftAKutsan please use upper case character after 'k' symbol

**Review by AKalinich-Luxoft — CHANGES_REQUESTED** [review]

File: src/components/application_manager/src/commands/hmi/rc_is_ready_request.cc [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    }
+    default: {
+      LOG4CXX_ERROR(logger_, "Received unknown event" << event.id());
+      return;
+    }
+  }
+}
+
+void RCIsReadyRequest::onTimeOut() {
+  // Note(dtrunov): According to new requirment APPLINK-27956
```
Comment (LitvinenkoIra) [inline src/components/application_manager/src/commands/hmi/rc_is_ready_request.cc]: @LuxoftAKutsan remove this comment pls

**Review by AKalinich-Luxoft — APPROVED** [review]

**Review by LitvinenkoIra — APPROVED** [review]

**Pull request merged**